specification/keyvault/Security.KeyVault.Certificates/client.tsp

```diff
@@ -14,6 +14,72 @@ namespace Customizations;
 
 using KeyVault;
 
+// Make optional path parameter required for all languages except Rust
+// TODO: have other languages eventually adopt optional path parameters
+
+/**
+ * The UpdateCertificate operation applies the specified update on the given certificate; the only elements updated are the certificate's attributes. This operation requires the certificates/update permission.
+ */
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Foundations.Operation is necessary for Key Vault"
+@summary("Updates the specified attributes associated with the given certificate.")
+@route("/certificates/{certificate-name}/{certificate-version}")
+@patch(#{ implicitOptionality: true })
+op updateCertificate is KeyVaultOperation<
+  {
+    /**
+     * The name of the certificate in the given key vault.
+     */
+    @path("certificate-name")
+    @clientName("name", "go")
+    certificateName: string;
+
+    /**
+     * The version of the certificate.
+     */
+    @path("certificate-version")
+    @clientName("version", "go")
+    certificateVersion: string;
+
+    /**
+     * The parameters for certificate update.
+     */
+    #suppress "deprecated" "Property flattening is supported for legacy scenarios like Key Vault's"
+    @body
+    @flattenProperty
+    parameters: CertificateUpdateParameters;
+  },
+  CertificateBundle
+>;
+
+/**
+ * Gets information about a specific certificate. This operation requires the certificates/get permission.
+ */
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Foundations.Operation is necessary for Key Vault"
+@summary("Gets information about a certificate.")
+@route("/certificates/{certificate-name}/{certificate-version}")
+@get
+op getCertificate is KeyVaultOperation<
+  {
+    /**
+     * The name of the certificate in the given vault.
+     */
+    @path("certificate-name")
+    @clientName("name", "go")
+    certificateName: string;
+
+    /**
+     * The version of the certificate. This URI fragment is optional. If not specified, the latest version of the certificate is returned.
+     */
+    @path("certificate-version")
+    @clientName("version", "go")
+    certificateVersion: string;
+  },
+  CertificateBundle
+>;
+
+@@override(KeyVault.updateCertificate, updateCertificate, "!rust");
+@@override(KeyVault.getCertificate, getCertificate, "!rust");
```

Comment:
Why? We treat these as required method parameters as well in line with all the other languages.

Reply:
Same for all packages herein.